charset.c:864:13: error: void function 'mimeheader_cat' should not return a value [-Wreturn-type]
--- lib/charset.c.orig	2011-10-04 14:59:14.000000000 -0500
+++ lib/charset.c	2014-03-30 01:45:25.000000000 -0500
@@ -861,7 +861,7 @@
     int len;
     char *res;
 
-    if (!s) return 0;
+    if (!s) return;
 
     /* set up the conversion path */
     input = table_init(0, target);
